nuclia/tests/test_manage/test_kb.py

```diff
@@ -0,0 +1,27 @@
+from uuid import uuid4
+
+from nuclia.sdk.kbs import NucliaKBS
+from nuclia.tests.fixtures import TESTING_ACCOUNT_SLUG, TESTING_KBID
+
+NEW_KB_SLUG = "testkb-" + uuid4().hex
+
+
+def test_list_kbs(testing_config):
+    kbs = NucliaKBS()
+    all = kbs.list()
+    assert TESTING_KBID in [kb.id for kb in all]
+
+
+def test_add_kb(testing_config):
+    kbs = NucliaKBS()
+    kb = kbs.add(account=TESTING_ACCOUNT_SLUG, slug=NEW_KB_SLUG, title="Test KB")
+    assert kb["id"] is not None
+    assert kb["slug"] == NEW_KB_SLUG
+    assert kb["title"] == "Test KB"
+
+
+def test_delete_kb(testing_config):
```

Comment:
If I am not wrong, this test depends on the one above.
I would recommend creating a new KB for the delete test, and make sure it is deleted.

Reply:
Fo simplicity sake, I have grouped the creation and the deletion test together.